packages/passport-interface/src/RequestTypes.ts

```diff
@@ -589,3 +637,47 @@ export type KnownTicketTypesRequest = undefined;
  * issue the user.
  */
 export const ISSUANCE_STRING = "Issue me PCDs please.";
+
+/**
+ * Collection of tickets that some clients keep track of so that the tickets
+ * contained within it function offline.
+ */
+export interface OfflineTickets {
+  devconnectTickets: OfflineDevconnectTicket[];
+  secondPartyTickets: OfflineSecondPartyTicket[];
+}
+
+/**
+ * New empty {@link OfflineTickets}.
+ */
+export function defaultOfflineTickets(): OfflineTickets {
+  return {
+    devconnectTickets: [],
+    secondPartyTickets: []
+  };
+}
+
+/**
+ * @todo - redact this? I think devconnect tickets are fine because
+ * only event staff have access to this.
+ */
```

Comment:
cc @artwyman @robknight

Reply:
I'm all for minimalism if not all of these fields are needed, but I agree that for the Devconnect "super user" use case, there's not a serious privacy concern.